deployments/ecs/ec2/README.md

```diff
@@ -0,0 +1,155 @@
+# Amazon ECS EC2 Deployment
+Familiarity with Amazon ECS using launch type EC2 is assumed. Consult the 
+[Getting started with the Amazon ECS console using Amazon EC2](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/getting-started-ecs-ec2.html)
+for further reading.
+
+The
+[Splunk OpenTelemetry Connector](https://github.com/signalfx/splunk-otel-collector)
+(Collector) should to be run as a Daemon service in an EC2 ECS cluster.
+
+Requires Connector release v0.33.0 or newer which corresponds to image tag 0.33.0 and newer.
+See image repository [here](https://quay.io/repository/signalfx/splunk-otel-collector?tab=tags).
+
+## Getting Started
+### Create Task Definition
+Take the task definition JSON for the Collector [here](./splunk-otel-collector.json), replace
+`MY_SPLUNK_ACCESS_TOKEN` and `MY_SPLUNK_REALM` with valid values. Update the image tag to
+the newest version. Use the JSON to create a task definition of **EC2 launch type** following
+the instructions [here](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/create-task-definition.html).
+
+The Collector is configured to use the default configuration file `/etc/otel/collector/ecs_ec2_config.yaml`.
+The Collector image Dockerfile is available [here](../../../cmd/otelcol/Dockerfile) and the contents of the default
+configuration file can be seen [here](../../../cmd/otelcol/config/collector/ecs_ec2_config.yaml).
+
+The configured network mode for the task is **host**. This means that **task metadata endpoint
+version 2** used by receiver `smartagent/ecs-metadata` will not be enabled by default. See
+[here](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/task-metadata-endpoint.html)
+for the task metadata endpoint version that will be enabled by default for your task.
+
+Add the following to the **environment** list in the task definition JSON if **task metadata
+endpoint version 3** will be enabled for your task:
```

Comment:
Not version 4 like the envvars used?

Reply:
No, version 3 the endpoint env var is ECS_CONTAINER_METADATA_URI whereas for version 4 it is ECS_CONTAINER_METADATA_URI_V4.